storage.bs

```diff
@@ -471,20 +471,21 @@ locally.
 
 <h2 id=usage-and-quota>Usage and quota</h2>
 
-<p>The <dfn export>storage usage</dfn> of a <a>storage shelf</a> is a rough estimate of the amount
-of bytes used by it.
+<p>The <dfn export>storage usage</dfn> of a <a>storage shelf</a> is an <a>implementation-defined</a>
+rough estimate of the amount of bytes used by it.
 
 <p class=note>This cannot be an exact amount as user agents might, and are encouraged to, use
 deduplication, compression, and other techniques that obscure exactly how much bytes a
 <a>storage shelf</a> uses.
 
-<p>The <dfn export>storage quota</dfn> of a <a>storage shelf</a> is a conservative estimate of the
-total amount of bytes it can hold. This amount should be less than the total available storage space
-on the device to give users some wiggle room.
+<p>The <dfn export>storage quota</dfn> of a <a>storage shelf</a> is an <a>implementation-defined</a>
+conservative estimate of the total amount of bytes it can hold. This amount should be less than the
+total storage space on the device. It must not be a function of the available storage space on the
+device.
```

Comment:
It'd be good to add a note explaining why.

Reply:
I think this paragraph change is similar to #106, which says "to avoid facilitating a cross-origin resource size leak"

Reply:
Good point, done.